test/components/Slider.test.jsx

```diff
@@ -3,7 +3,11 @@ import renderer from 'react-test-renderer';
 import 'raf/polyfill';
 import Slider from '../../src/components/Slider';
 
-it('renders correctly', () => {
+// Skipping this for now as we get "TypeError: Cannot read property
+// 'addEventListener' of null" as react-test-renderer doesn't work with refs
+// well. https://material-ui.com/guides/migration-v3/#core
+// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Can use enzyme instead, as suggested by various issues.

Reply:
Yes, please 👍

Reply:
So I tried using enzyme, turned out to be uglier than expected. I was able to reduce snapshot size using `createSerializer.map` in `jest.config.js`, but it's still ugly as I couldn't figure out how to remove those `WithStyles` etc and render only native html elements. Removing props from `WithStyles` etc. made snapshots much cleaner and less brittle though.

Reply:
Tried again, probably hacky way - but works.

Reply:
If it makes your life easier you can use React testing library https://testing-library.com/docs/react-testing-library/intro

We've been moving toward it on our Treeherder project.

Reply:
Could be a good option. Would be better to skip snapshots and write _actual_ tests maybe?

Reply:
Snapshots look much better :)

Reply:
I'm fine with that.

Here are some examples of using the testing library:
https://github.com/mozilla/treeherder/search?q=testing-library&unscoped_q=testing-library